charts/kong/UPGRADE.md

```diff
@@ -45,6 +45,32 @@ clear it.
 
 ## 1.4.0
 
+### Changes to default Postgres permissions
+
+The [Postgres sub-chart](https://github.com/bitnami/charts/blame/master/bitnami/postgresql/)
+used by this chart has modified the way their chart handles file permissions.
+This is not an issue for new installations, but prevents Postgres from starting
+if its PVC was created with an older version. If affected, your Postgres pod
+logs will show:
+
+```
+postgresql 19:16:04.03 INFO  ==> ** Starting PostgreSQL **
+2020-03-27 19:16:04.053 GMT [1] FATAL:  data directory "/bitnami/postgresql/data" has group or world access
+2020-03-27 19:16:04.053 GMT [1] DETAIL:  Permissions should be u=rwx (0700).
+```
+
+You can restore the old permission handling behavior by adding two settings to
+the `postgresql` block in values.yaml:
+
+```
+postgresql:
+  postgresqlDataDir: /bitnami/postgresql/data
```

Comment:
Can we please add `enabled: true` before this line.
Users can and will copy pasta this and wonder why they have a whole new set of problems now.